test/verify/check-system-services

```diff
@@ -34,7 +34,6 @@ class TestServices(MachineCase):
         # when we add additional services to this target below
         # we don't race with the boot process
         self.machine.execute("systemctl start default.target")
-        self.machine.execute("systemctl --user is-active default.target")
```

Comment:
Bad rebase, I tried to drop that commit. The issue is that at this stage there is no user session - which also points to those bogus `systemctl --user` calls. But I'll check the system one to do waiting instead of starting.